### PR TITLE
[Method References 2] LPS-171146 Convert method reference to ServiceTrackerMap in module portal-workflow-kaleo-runtime-impl | part 2

### PR DESCRIPTION
**Pull request by W2jx**

### CODE REVIEW

**Comment by W2jx** [comment]
ci:test:workflow

**Comment by Kvale1733** [comment]
@W2jx No regressions found.

**Comment by W2jx** [comment]
no conflicts(2022/12/16 8:47)

**Comment by W2jx** [comment]
no conflicts(12/19 9:40)

**Comment by W2jx** [comment]
no conflicts（12/20 11:57）

**Comment by W2jx** [comment]
no conflicts(12/21 17:05)

File: modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/notification/NotificationMessageGeneratorFactory.java [hunk 1]
```diff
@@ -48,43 +45,19 @@ public NotificationMessageGenerator getNotificationMessageGenerator(
...
-		Map<String, Object> properties) {
-
-		String[] templateLanguages = _getTemplateLanguages(properties);
-
-		for (String templateLanguage : templateLanguages) {
-			_notificationMessageGenerators.remove(templateLanguage);
-		}
+	@Activate
+	protected void activate(BundleContext bundleContext) {
+		_serviceTrackerMap = ServiceTrackerMapFactory.openSingleValueMap(
```
Comment (tinatian) [inline modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/notification/NotificationMessageGeneratorFactory.java]: @W2jx The logic is not consistent before and after your change

**Comment by tinatian** [comment]
@W2jx Please resend.

Thanks.
Tina

**Pull request closed without merging**